ricardo/access_modifiers/README.md

```diff
@@ -0,0 +1,43 @@
+# Access Modifiers
+There are 4 basics `access modifiers`, global, class, instance and local, all this are defined by a
+prefix, here are the prefixes:
+
+- Global: `$`.
+- Class: `@@`.
+- Instance: `@`.
+- Local: `_` or nothing.
+
+## Scopes
+- Global: everything.
+- Class: only accessible within the class.
+- Instance: only the instance, from the instance and within the instance class.
+- Local: only within the current context.
+
+## Example
+```ruby
+$global_var = 'Hello from outside'
+
+class Person
+  def initialize
+    ## heres if in any instance @@greet is modified, all instances will get the same value from this
+    variable
+    @@greet = 'greeting from class (everyone should get the same info)'
+    @message = 'This is unique for instance'
+  end
+
+  def self.greet
```

Comment:
Class methods should be separated from instance methods, usually writen before initialize